test/unit/lib/plugins/aws/config-credentials.test.js

```diff
@@ -110,26 +110,26 @@ describe('AwsConfigCredentials', () => {
   });
 
   describe('#configureCredentials()', () => {
-    it('should lowercase the provider option', () => {
+    it('should lowercase the provider option', async () => {
       awsConfigCredentials.options.provider = 'SOMEPROVIDER';
 
       return awsConfigCredentials.configureCredentials().then(() => {
         expect(awsConfigCredentials.options.provider).to.equal('someprovider');
       });
     });
 
-    it('should use the "default" profile if option is not given', () =>
+    it('should use the "default" profile if option is not given', async () =>
       awsConfigCredentials.configureCredentials().then(() => {
         expect(awsConfigCredentials.options.profile).to.equal('default');
       }));
 
-    it('should resolve if the provider option is not "aws"', (done) => {
+    it('should resolve if the provider option is not "aws"', async () => {
       awsConfigCredentials.options.provider = 'invalid-provider';
 
-      awsConfigCredentials.configureCredentials().then(() => done());
+      expect(awsConfigCredentials.configureCredentials()).to.be.eventually.fulfilled;
```

Comment:
I think we need to `return` here

Reply:
sure, let me fix this.
while I was going through the tests with promise, I notice that they are written either use `await` or return promise. Maybe we can decide on one and consolidate from now on.

Reply:
Probably `await` is cleaner, but `return` is totally acceptable as well